pretyping/tacred.ml

```diff
@@ -267,9 +267,27 @@ let invert_name labs l {binder_name=na0} env sigma ref na =
    [compute_consteval_mutual_fix] only one by one, until finding the
    last one before the Fix if the latter is mutually defined *)
 
+let whd_betadeltazeta_never_unfold env s =
+  let flags =
+    let open CClosure.RedFlags in
+    let open ReductionBehaviour in
+    red_add
+      (red_add
+         (List.fold_right
+            (fun x r ->
+              red_sub r
+                (match x with
+                   Names.GlobRef.ConstRef xx ->  fCONST xx
```

Comment:
This deserves a comment.
Also, I'm not 100% sure that one cannot use `Arguments` on a `Let` for example, which is a `VarRef` IIRC

Reply:
Two answers:
1/ from the code, it is not possible that a "Let" defined constant has a flag stored in the ReductionBehaviour table, because this only works for constants.
2/ I also tried.  the following code fragment
```
Section playground.
Let plus3 := fix plus3 (n m : nat) := match n with O => m | S p => plus3 p (S m) end.
Arguments plus3 : simpl never.
```
got the following error message.
```
Error: Modifiers of the behavior of the simpl tactic are relevant for
constants only.
```

Reply:
ok!